Client/Frontend/Home/FirefoxHomeContextMenuHelper.swift

```diff
@@ -60,6 +82,18 @@ class FirefoxHomeContextMenuHelper: HomePanelContextMenu {
         }.items
     }
 
+    // MARK: - History Highlights
+
+    private func getHistoryHighlightsActions(for highlightItem: HighlightItem) -> [PhotonRowActions]? {
+        return [SingleActionViewModel(title: .RemoveContextMenuTitle,
+                                     iconString: ImageIdentifiers.actionRemove,
+                                     tapHandler: { _ in
+
+            self.viewModel.historyHighlightsViewModel.delete(highlightItem)
+            self.sendTopSiteContextualTelemetry(type: .remove)
```

Comment:
shouldn't call to sendHistoryHighlightContextualTelemetry? maybe a copy/paste error